### DISCUSSION TITLE
[Feature] JPEG XL HDR support

### DISCUSSION THREAD

**Discussion by f-ij** [immich-app/immich#12890] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Since lightroom supports the exporting of HDR images now, I export images from my camera in JXL with HDR. However, these images don't show properly either through the web interface on my Mac or the app.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Dup #8284
